safe-global/safe-wallet-web
# fix: add {RecoveryHub} logo

## What it solves

Resolves outdated logo

## How this PR fixes it

The correct logo has been added according to [the designs](https://www.figma.com/file/pO43pjVwR34Uy1ihMsY3OM/Generic-recovery?type=design&node-id=1583-40673&mode=design&t=7pzyIwna28ihtzgN-0) and adjusted for dark mode.

## How to test it

Open a Safe on a chain that supports recovery, ensuring that it is _not_ enabled. Observe the new logo on the dashboard that should change with dark mode.

## Screenshots

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/1a00d22f-5ec2-4077-bc07-929a39583f5b)

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/12f21827-b352-49af-b1c7-c483bf09300b)

## Checklist
